public/server/db.json

```
@@ -109,6 +109,57 @@
     },
     "ootd":{
         "definition": "Outfit Of The Day"
-    }
-    
+    },
+    "cmiiw":{
+        "definition": "Correct me if i'm wrong"
```

These all look AMAZING! Each letter of each word in the definition has to be uppercase though. Please look over your additions again. 
```suggestion
        "definition": "Correct Me If I'm Wrong"
```